skills/review-judgment/scripts/Get-AcRefsFromIssue.ps1

```
@@ -0,0 +1,47 @@
+    # Isolate the ## Acceptance Criteria section (case-insensitive, multiline).
+    $parts = $body -split "(?im)^##\s+acceptance criteria\s*$", 2
+    if ($parts.Count -lt 2) { return @() }
+    $acSection = $parts[1]
+
+    # Cut off at the next H2 header so we don't leak into later sections.
+    $acSection = ($acSection -split "(?m)^##\s", 2)[0]
+
+    # Extract backtick-quoted file paths with code-relevant extensions.
+    $extensionPattern = 'ps1|md|json|yml|yaml|xml|sql|ts|tsx|js|jsx|py|cs|java|go|rs'
+    $regex = "``([a-zA-Z0-9_\-./]+\.(?:$extensionPattern))``"
+    $matches = [regex]::Matches($acSection, $regex)
```

PSScriptAnalyzer (1.25.0)

[warning] The Variable 'matches' is an automatic variable that is built into PowerShell, assigning to it might have undesired side effects. If assignment is not by design, please use a different name.

(PSAvoidAssignmentToAutomaticVariable)